[2022-07-18 23:46:53,088] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2004-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:46:53,097] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2004-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:46:53,098] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:46:53,099] {taskinstance.py:1358} INFO - Starting attempt 2 of 2
[2022-07-18 23:46:53,100] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:46:53,112] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2004-01-10 06:00:00+00:00
[2022-07-18 23:46:53,121] {standard_task_runner.py:52} INFO - Started process 40175 to run task
[2022-07-18 23:46:53,125] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'local_to_gcs', 'local_to_gcs_task', 'scheduled__2004-01-10T06:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/local_to_gcs.py', '--cfg-path', '/tmp/tmpl20tx0kw', '--error-file', '/tmp/tmp_azcdp2a']
[2022-07-18 23:46:53,128] {standard_task_runner.py:80} INFO - Job 76: Subtask local_to_gcs_task
[2022-07-18 23:46:53,192] {task_command.py:370} INFO - Running <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2004-01-10T06:00:00+00:00 [running]> on host 32d1f26135d5
[2022-07-18 23:46:53,263] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_to_gcs
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2004-01-10T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2004-01-10T06:00:00+00:00
[2022-07-18 23:46:59,177] {python.py:173} INFO - Done. Returned value was: None
[2022-07-18 23:46:59,212] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=local_to_gcs, task_id=local_to_gcs_task, execution_date=20040110T060000, start_date=20220718T234653, end_date=20220718T234659
[2022-07-18 23:46:59,281] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-18 23:46:59,335] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
